### ISSUE TITLE
implement method, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1561]
```abap
INTERFACE lif_repo.
  METHODS has_remote_source.
ENDINTERFACE.

INTERFACE lif_online.
  INTERFACES lif_repo.
  ALIASES has_remote_source FOR lif_repo~has_remote_source.
ENDINTERFACE.

CLASS lcl_repo DEFINITION ABSTRACT.
  PUBLIC SECTION.
    INTERFACES lif_repo ABSTRACT METHODS has_remote_source.
    ALIASES has_remote_source FOR lif_repo~has_remote_source.
ENDCLASS.

CLASS lcl_repo IMPLEMENTATION.
ENDCLASS.

CLASS lcl_online DEFINITION INHERITING FROM lcl_repo.
  PUBLIC SECTION.
    INTERFACES lif_online.
    METHODS has_remote_source REDEFINITION.
ENDCLASS.

CLASS lcl_online IMPLEMENTATION.
  METHOD has_remote_source.
    RETURN.
  ENDMETHOD.
ENDCLASS.
```

![image](https://user-images.githubusercontent.com/5888506/99981111-45e09080-2da9-11eb-8418-443ab9496cc3.png)
